docs/release-notes-pxc-8.4.2.md

```diff
@@ -0,0 +1,25 @@
+# Percona Distribution for MySQL 8.4.2 using Percona XtraDB Cluster (2024-12-12)
+
+Percona Distribution for MySQL is the most stable, scalable, and secure open source MySQL distribution, with two download options: one based on Percona Server for MySQL and one based on Percona XtraDB Cluster. [Install Percona Distribution for MySQL](installing.md).
+
+This release is based on [Percona XtraDB Cluster 8.4.2-2](https://docs.percona.com/percona-xtradb-cluster/8.4/release-notes/8.4.2-2.html).
+
+## Release highlights
+
+Percona XtraDB Cluster is based on Percona Server for MySQL. Find a complete list of improvements and bug fixes in the [Percona Server for MySQL 8.4.2-2 (2024-11-04) release notes](https://docs.percona.com/percona-server/8.4/release-notes/8.4.2-2.html).
+
+## Known issues
+
+* This release does not include the Percona Toolkit component. We will add it once a Percona Toolkit version compatible with MySQL 8.4 becomes available.
```

Comment:
A component has a specific definition in MySQL. 
Just say 
include the Percona Toolkit. We will add the Percona Toolkit when a version compatible with MySQL 8.4 becomes available.

Reply:
updated